<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationForm</class>
 <widget class="QMainWindow" name="RegistrationForm">
  <property name="geometry">
   <rect>
    <int>200</int>
    <int>200</int>
    <int>400</int>
    <int>300</int>
   </rect>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="username_label">
      <property name="text">
       <string>Логин:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="username_input"/>
    </item>
    <item>
     <widget class="QLabel" name="password_label">
      <property name="text">
       <string>Пароль:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="password_input">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="phone_label">
      <property name="text">
       <string>Телефон:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="phone_input"/>
    </item>
    <item>
     <widget class="QPushButton" name="register_button">
      <property name="text">
       <string>Зарегистрироваться</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="result_label">
      <property name="text">
       <string></string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
</ui>
